cli/script/command-executor.ts

```diff
@@ -261,9 +271,19 @@ function deserializeConnectionInfo(): IConnectionInfo {
     return tryJSON(json);
 }
 
+function alreadyLoggedIn(): Promise<void> {
+     return Promise<void>((resolve, reject, notify): void => {
+         log("You are already logged in from this machine!");
+     });
+}
+
 export function execute(command: cli.ICommand): Promise<void> {
     connectionInfo = deserializeConnectionInfo();
 
+    if (connectionInfo && command.type === cli.CommandType.login) {
```

Comment:
Yes, earlier I had error message shown for register too, but I realized that it isn't really needed since you could just run register multiple times if for some reason you want to have different provider configured. Moving this to switch block.